### DISCUSSION TITLE
Awesome library!

### DISCUSSION THREAD

**Discussion by anqit** [flyerhq/flutter_chat_ui#195] [Feedback]
I'm super pumped this exists, thanks for everyone who contributed to this open source library. Managed alternatives are incredibly costly, prohibitively so for startups and POCs, so something open source like this that provides a large portion of the functionality for none of the cost is really cool to see.

**Comment by demchenkoalex** [comment]
Thank you very much! Also great when people understand this is not a Facebook Messenger/Telegram level, but rather something to quickly come up with POC or launch a small startup :)

**Reply by anqit** [reply]
There's no reason it can't get there eventually with community support! 😄
